api-playground/openapi/setup.mdx

```diff
@@ -9,43 +9,61 @@ To describe your endpoints with OpenAPI, make sure you have a valid OpenAPI docu
 
 There are two ways to configure your docs to use your OpenAPI file:
 
-### In the repo
+**1) In the repo**: The most common method of setting up OpenAPI with Mintlify is to add your OpenAPI documents directly to your docs repo. The format will either be in `.yaml` or `.json`. 
 
-The most common method of setting up OpenAPI with Mintlify is to add your OpenAPI documents directly to your docs repo. You'll then add the paths to each OpenAPI document to the `openapi` field in your `mint.json`:
+**2) On the web**: If you have a publicly-hosted OpenAPI document that serves as the single-source-of-truth for your API, you can use that in your docs through the document URL (ex: https://mintlify.com/specs/v2/openapi.json)
```

Comment:
I think it would be nice to use our accordions for this. This way, the user can select which one makes more sense?

Reply:
Added them